src/service/discovery.js

```diff
@@ -178,14 +178,29 @@ const fetchInitialDiscoveryTopics = async (limit = 50, page = 0) => {
  * @property {FetchInitialDiscoveryUsersResponse.SuggestedUsers[]} [suggestedUsers]
  */
 
+/**
+ * @typedef {Object} FetchInitialDiscoveryUsersOptionalParams
+ * @property {string[]} excudedUserIds
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a typo in the property name `excudedUserIds`. It should be `excludedUserIds`.
```diff
-  * @property {string[]} excudedUserIds
+  * @property {string[]} excludedUserIds
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->